src/main/webapp/app/shared/util/core-evidence-submission/core-evidence-submission.spec.ts

```diff
@@ -456,6 +459,7 @@ describe('getEvidence to submit to core', () => {
                     prognosticSummary_uuid: 'd9c991e4-1c13-4251-bd5a-fac0344b3470',
                     prognosticSummary: 'Prognostic Summary',
                     prognosticSummary_review: createMockReview({
+                      lastReviewed: 'Prognostic Summary Last Reviewed',
```

Comment:
What's the rational for testing Last Reviewed for only prognostic?

Reply:
I felt like if I tried to test every case it'd get too complicate, but as far as why specifically prognostic summary? There isn't a real reason.

Reply:
Let's pick few relatively important ones? Could you do oncogenicity/Therapeutic Summary/Tx level